packages/flutter_tools/test/commands.shard/hermetic/devices_test.dart

```diff
@@ -66,3 +79,12 @@ class MockProcessManager extends Mock implements ProcessManager {
     return ProcessResult(0, 0, '', '');
   }
 }
+
+class _MockDeviceManager extends DeviceManager {
```

Comment:
nit: renamed to MockDeviceManager

Reply:
I guess it should also become `FakeDeviceManager`

Reply:
Yes, if its not extends Mock it is a Fake

Reply:
@jonahwilliams I guess it's ready for review